docs/source/json-api/index.rst

```diff
@@ -92,21 +92,22 @@ using.
 With Query Store
 ------------------
 
-In production setups, you should configure the JSON API to use a
-PostgreSQL backend as a cache. The in-memory backend will call the
+You can configure the JSON API to use a PostgreSQL/Oracle(Enterprise Edition Only)
```

Comment:
The statement about this relevant only for production setups seems relevant, I'd keep it in.